devbox/install

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ -z $( which devbox ) ]; then
+    echo "Installing devbox"
+    curl -fsSL https://get.jetify.com/devbox | bash
```

Comment:
kriegen wir das noch versioniert und mit in renovate reingehangen? Damit wir alle das gleich haben.

Reply:
Bring nichts, wird ja nur einmalig bei der Installation ausgeführt. Danach würde ein `devbox global update` das Update durchführen